client/landing/stepper/router/test/index.tsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import { Flow, From, To, BackTo, createNavigator } from '../index';
+
+interface FlowContext {
+	siteSlug: string;
+}
+
+describe( 'Stepper Router', () => {
+	const positiveCondition = () => true;
+	const negativeCondition = () => false;
+
+	it( 'return all the router steps', () => {
+		const result = createNavigator(
+			<Flow name="test" persistParam={ [ 'from', 'to', 'siteSlug' ] }>
+				<From step="STEP_1">
+					<To step="STEP_2A" when={ positiveCondition } />
```

Comment:
It would be nice to see it applied in a real flow, so we could figure out more things. I'm thinking if the code to create the booleans would be simple or we would need a complex code similar to our existing flows to create these `when` variables.

Reply:
Yes, on my next free time I will aim to apply.